src/apps/weblib/interface.cpp

```diff
@@ -356,11 +356,7 @@ void Interface::setLogging(bool enable)
 }
 
 void Interface::update(ObjectRegistry::Handle chart,
-    ObjectRegistry::Handle canvas,
-    double width,
-    double height,
-    double timeInMSecs,
-    bool render)
+    double timeInMSecs)
```

Comment:
Make the C++ interface parameter to C++ class (`std::chrono::duration<double, std::milli>`)

Reply:
Currently we have the C->C++ type conversions in interface.cpp. The idea was that this is a C interface but the implementation is encapsulated in a c++ class, and the standalone functions are just thin wrappers. If we want to change the concept, we should move all type conversations to cinterface.cpp

Reply:
I previously changed the `setChartFilter`, `getRecordValue`, and `set/get AnimControlValue` (string_view). I think this concept can be reached step-by-step with small modifications.

Reply:
Let's do this in one step, it's more efficient: 
interface.cpp method arguments should be c++ types
[#513](https://github.com/vizzuhq/vizzu-lib/issues/513)